concordia/static/scss/action-app.scss

```diff
@@ -144,32 +228,60 @@ main {
             width: 100%;
             height: 100%;
         }
+
+        #toolbar {
+            order: 1;
+        }
+        #help-panel {
+            order: 3;
+        }
+        #grid-toggle {
+            display: block;
+        }
     }
 
     &:not([data-open-asset-id]) #close-viewer-button {
         display: none;
     }
 }
 
-#asset-info-modal {
-    &.modal.show {
-        width: 100vw;
-        .modal-dialog {
-            max-width: 90vw;
+#toolbar {
```

Comment:
This should have a name which describes what it applies to in more detail, especially since it's declared as a  global ID